libraries/AP_Baro/AP_Baro.h

```diff
@@ -177,6 +180,8 @@ class AP_Baro
         float altitude;                 // calculated altitude
         AP_Float ground_pressure;
         float p_correction;
+        uint8_t can_manager;
+        uint8_t uavcan_node;
```

Comment:
I still don't get it. Let's take it step by step and you'll tell me where I'm wrong:

- each CAN manager has one UAVCAN instance
- the place where it is checked if a baro has already been registered is at https://github.com/ArduPilot/ardupilot/pull/6167/files#diff-edf025cca2b87dd009d5612c806aa979R630
- the checks are `sensors[i].can_manager == mgr` and `sensors[i].uavcan_node == node`, meaning that a baro is deemed as registered only if it came from the same manager and has the same node number
- if the baro is only already registered if it came from the same manager, how is it not possible for AP_UAVCAN to keep a list of nodes registered when each UAVCAN instance corresponds to one manager?